manifest.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "LBCAlerte",
-    "id": "LBCAlerte",
+    "name": "Cheky",
+    "id": "Cheky",
```

Comment:
If you change the id, beyond the fact that existing users won't be able to upgrade (which isn't a problem in my opinion), you'd better fork the repository to create a cheky_ynh repository. We could then add it to the YunoHost-Apps organization... you could even becom e the maintainer if you wish :smile: 
Moreover, id's should stay in lowercases (rule which I didn't respect in 2016 :wink:)